debian/ubuntu-advantage-tools.postinst

```diff
@@ -318,6 +319,28 @@ enable_periodic_license_check() {
     fi
 }
 
+disable_new_timer_if_old_timer_already_disabled() {
+    # If the user has disable the old timer (ua-messaging)
+    # We will assume that the user would also not want the
+    # new timer to be disabled as well. Therefore, we will be
+    # manually disabling the new timer here.
+    PREVIOUS_PKG_VER=$1
+
+    # We should only perform this check on UA version that have the
+    # ua-messaging.timer: 27.0 until 27.2. This will also guarantee
+    # that on 27.3 and forward, we will not run this logic.
+    if dpkg --compare-versions "$PREVIOUS_PKG_VER" ge-nl "27.0~"; then
+        if dpkg --compare-versions "$PREVIOUS_PKG_VER" lt-nl "27.3~"; then
+            if [ -x "/usr/bin/deb-systemd-helper" ]; then
+                if ! deb-systemd-helper --quiet was-enabled $OLD_MESSAGING_TIMER; then
+                    systemctl stop $NEW_TIMER
+                    systemctl mask $NEW_TIMER
```

Comment:
Can't wait until we add another timer and then we'll have a variable named `$NEW_NEW_TIMER` :upside_down_face: 

Just joking :) This looks good!

Reply:
We should consider adding these version if statements on all future postinst migrations too.